src/plotting/plotWindow.component.tsx

```diff
@@ -70,6 +71,8 @@ const PlotWindow = (props: PlotWindowProps) => {
   const [rightYAxisScale, setRightYAxisScale] = React.useState<YAxisScale>(
     plotConfig.rightYAxisScale
   );
+  const [leftYAxisLabel, setLeftYAxisLabel] = React.useState('');
+  const [rightYAxisLabel, setRightYAxisLabel] = React.useState('');
```

Comment:
These aren't initialising from the `plotConfig`, so even though the axis labels are saving to Redux, when you reopen a plot after saving the labels aren't showing up.